### ISSUE TITLE
Have you ever test the cifar10 with shufflenet v2 and what's the result?

### ISSUE THREAD

**Issue by agdyangkang** [kuangliu/pytorch-cifar#81]

**Comment by ngcthuong** [comment]
I just tested it recently but with a different learning rate of 0.1, 0.01, and 0.001 for 25, 50, 100 (totally 100 epochs)

ShuffleNetG2 - 90.32%  
ShuffleNetG3 - 
ShuffleNetV2 - 90.71%

**Comment by ajkdrag** [comment]
I was able to achieve 0.94 (with ema) and 0.938 (without ema) using ShuffleNetV2. The training is really slow though.
